google-cloud-bigtable-emulator/src/test/java/com/google/cloud/bigtable/emulator/v2/ITBigtableEmulatorRuleTest.java

```diff
@@ -38,7 +38,7 @@
 import org.junit.runners.JUnit4;
 
 @RunWith(JUnit4.class)
-public class BigtableEmulatorRuleTest {
+public class ITBigtableEmulatorRuleTest {
```

Comment:
Why does this need to be changed into an integration test?

Reply:
The native profile in java-shared-config (as [seen here](https://github.com/googleapis/java-shared-config/blob/b5e05c6726c60a7ad7c0d9588b4fab69a0b4e6b7/pom.xml#L811)) enables tests for native image testing based on the naming conventions: `IT*` and `*ClientTest`. This is why we needed to rename this to be picked up by the profile.

Reply:
doesnt this mean that we need to change all of tests in this entire project for them to be run by graal? Also, this needs to be documented so that when we add tests we use the correct naming scheme